src/pages/cluster-management/clusterInfo.html

```diff
@@ -18,139 +20,92 @@ <h1>
      size="75">
 </div>
 
-<div ng-hide="loader" class="clearfix mb-2">
-    <button type="button" class="btn btn-secondary delete-cluster-btn mr-2" ng-if="clusterConfiguration"
+<div ng-hide="loader" class="clearfix mb-2" ng-if="clusterConfiguration">
```

Comment:
Using ng-if and ng-hide on same node is confusing

Reply:
Ah.. yes it is. I'm just not used to having a few ways of showing/hiding stuff. Removed the `ng-hide` directive.